www/include/monitoring/comments/listComment.php

```diff
@@ -151,7 +156,12 @@
         $tab_comments_svc[$i]['data'],
         ['a', 'br', 'hr']
     );
+    $tab_comments_svc[$i]['h_details_uri'] = $resourceController->buildHostDetailsUri($data['host_id']);
     if ($data['service_description'] != '') {
+        $tab_comments_svc[$i]['s_details_uri'] = $resourceController->buildServiceDetailsUri(
```

Comment:
If the variable `$tab_comments_svc[$i]['host_name_link']` is no longer used, don't forget to delete it.